docs/source/index.rst

```diff
@@ -95,3 +95,4 @@ test
 .. autofunction:: print_training_update
 .. autofunction:: print_test_update
 
+.. mdinclude:: ../../OP_LOWERING_GUIDE.md
```

Comment:
I didn't know about this 😄 ... we have other `.md` files, should we include them explicitly @ailzhang ?

Reply:
Aha it's also the first time I learned about it. If it works, we should definitely add other mds as well :P

Reply:
Let me merge this one in and see if works. If it does we can discuss about which documents we wanted to add and in what order 😄